public/app/features/dashboard/components/PubdashFooter/PubdashFooter.tsx

```diff
@@ -11,7 +11,7 @@ export const PubdashFooter = function () {
     <div className={styles.footer}>
       <span className={styles.logoText}>
         <a href="https://grafana.com/" target="_blank" rel="noreferrer noopener">
-          powered by Grafana <img className={styles.logoImg} src="public/img/grafana_icon.svg"></img>
+          powered by Grafana <img className={styles.logoImg} alt="Grafana logo" src="public/img/grafana_icon.svg"></img>
```

Comment:
nit: 🤔 I am not sure we should have an `alt` text for the grafana logo, technically the "Powerered by Grafana" is communicating properly that this is a link point to the grafana website.